cli/src/config/windows.toml

```diff
@@ -1,2 +1,3 @@
 [ui]
```

Comment:
Perhaps add a note to `config.md` explaining that pagination is temporarily disabled?

Reply:
I had actually added it there but then changed my mind because most users seem to use a released version. But it's better to warn that it's disable when it's not than the other way around. I've added back that sentence.

Reply:
Good point. Feel free to revert the change.

Reply:
We should keep a list of all the doc changes blocked on being able to send users to the docs for the released version.

Reply:
Actually, I'll resolve this conversation again, in case you just want to submit this. I think either way is fine.